package/Runtime/Scripts/Nimbus.Internal/Network/NimbusClient.cs

```diff
@@ -23,7 +23,8 @@ public class NimbusClient {
 
 		public NimbusClient(CancellationTokenSource ctx, NimbusSDKConfiguration configuration) {
 			Client.DefaultRequestHeaders.Add("Nimbus-Api-Key", configuration.apiKey);
-			Client.DefaultRequestHeaders.Add("Nimbus-Sdkv", "1.0.0");
+			Client.DefaultRequestHeaders.Add("Nimbus-Sdkv", "2.1.0");
```

Comment:
If ever the Native SDKs diverge in version Nimbus, I'll move this to a project json file so that the Editor and Runtime assembly share a single referece